File: src/test/java/TC/저주의숫자3Test.java
package TC;

import org.assertj.core.api.Assertions;
import org.example.TC.저주의숫자3;
import org.junit.jupiter.api.Test;

public class 저주의숫자3Test {

    private final 저주의숫자3 curse = new 저주의숫자3();

    @Test
    void solution() {
        int x3 = curse.solution(15);
        int answer = 25;
        Assertions.assertThat(x3).isEqualTo(answer);
    }
}

File: src/test/java/TC(K)/저주의숫자3Test.kt
package `TC(K)`

import io.kotest.core.spec.style.StringSpec
import org.assertj.core.api.Assertions
import org.example.TCK.저주의숫자3

class 저주의숫자3Test:StringSpec(
    {
        "저주의 숫자 3test" {
            val n:Int = 15
            val result:Int = 25
            val answer:Int = 저주의숫자3().solution(n)

            Assertions.assertThat(result).isEqualTo(answer)
        }
    }
)
